### PR TITLE
RHIROS-653 rename account_id to tenant_id, update db queries

### PR DESCRIPTION
**Pull request by r14chandra**
Merge after this PR - #227

### CODE REVIEW

File: ros/api/v1/call_to_action.py [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+    identity, system_ids_by_org_id)
 from ros.lib.models import (
     PerformanceProfile, db)
 
 
 class CallToActionApi(Resource):
     def get(self):
         account_number = identity(request)['identity']['account_number']
-        system_query = system_ids_by_account(account_number).filter(PerformanceProfile.number_of_recommendations > 0)
+        system_query = system_ids_by_org_id(account_number).filter(PerformanceProfile.number_of_recommendations > 0)
```
Comment (kgaikwad) [inline ros/api/v1/call_to_action.py:12]:
```suggestion
        system_query = system_ids_by_org_id(org_id).filter(PerformanceProfile.number_of_recommendations > 0)
```

Method  `system_ids_by_org_id` accepts `org_id` whereas you passed `account_number`. 
Similarly here also you need to fetch `org_id` value from identity header?

File: ros/api/v1/hosts.py [hunk 2]
```diff
@@ -57,12 +57,12 @@
 
 class IsROSConfiguredApi(Resource):
     def get(self):
-        account_number = identity(request)['identity']['account_number']
+        org_id = identity(request)['identity']['internal']['org_id']
```
Comment (kgaikwad) [inline ros/api/v1/hosts.py:60]:
```suggestion
        org_id = org_id_from_identity_header(request)              
```

Almost everywhere ros-backend is parsing and fetch `org_id` value using nested keys fron identity header. 
Will it be good idea to encapsulate ` identity(request)['identity']['internal']['org_id']` logic into a new method [here](https://github.com/RedHatInsights/ros-backend/blob/main/ros/lib/utils.py).
After promoting `org_id` field as top level field in identity header, one needs to update code ONLY at one place.

**Review by kgaikwad — CHANGES_REQUESTED** [review]
Thank you @r14chandra. Added two inline comments!

File: ros/api/v1/call_to_action.py (hunk 1 above)
Comment (r14chandra) [reply ros/api/v1/call_to_action.py:12]: Oh I missed this 🤦🏻‍♀️.

File: ros/api/v1/hosts.py (hunk 2 above)
Comment (r14chandra) [reply ros/api/v1/hosts.py]: Yeah good idea 👍🏻

Comment (kgaikwad) [reply ros/api/v1/hosts.py]: @r14chandra  - Please go through slack conversation - https://ansible.slack.com/archives/C03B7U5AE67/p1655120034820549?thread_ts=1655111676.332289&cid=C03B7U5AE67
This change i.e org_is as top level field is already available in stage. 
Could you also use top level key to get org_id change?

File: ros/api/v1/recommendations.py [hunk 3]
```diff
@@ -51,9 +51,9 @@ def get(self, host_id):
 
         filter_description = request.args.get('description')
 
-        account_query = db.session.query(RhAccount.id).filter(RhAccount.account == ident['account_number']).subquery()
+        tenant_query = db.session.query(RhAccount.id).filter(RhAccount.org_id == ident['internal']['org_id']).subquery()
```
Comment (kgaikwad) [inline ros/api/v1/recommendations.py:54]:
```suggestion
        tenant_query = db.session.query(RhAccount.id).filter(RhAccount.org_id == ident['org_id']).subquery()
```

**Review by kgaikwad — APPROVED** [review]
Thanks @r14chandra  for resolving all comments.
Changes look good to me.

**Pull request merged**